transit.md

```diff
@@ -48,7 +48,8 @@ This may be relaxed in the future, much as Wormhole was.
 
 Each Transit object has a set of "abilities". These are outbound connection
 mechanisms that the client is capable of using. The basic CLI tool (running
-on a normal computer) has two abilities: `direct-tcp-v1` and `relay-v1`.
+on a normal computer) has these abilities: `direct-tcp-v1`, `relay-v1`,
+`tor-tcp-v1`, `direct-ws-v1` and `direct-wss-v1`.
```

Comment:
I feel like these should be called `relay-ws-v1` and `relay-wss-v1` but I also feel like we had this conversation before and I don't recall how it ended. Could you remind me what's up with that? 

If we did go with the above, would it then make sense to rename `relay-v1` to `relay-tcp-v1` (in a backwards compatible way w/ deprecation warning)?

Perhaps we can add descriptions of these in the list that follows? Maybe something like "indicates that it can connect via the transit relay using <proto> transport".